styles/styles.css

```
@@ -0,0 +1,539 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,400;0,700;1,400&family=Roboto:ital,wght@0,400;0,700;1,400&display=swap');
+
+/*////////////////////////////////////////////////////////
+table of contents
+1. global styles
+2. typography
+3. areas
+    3.1 header
+    3.2 main
+    3.3 footer
+////////////////////////////////////////////////////////
+Paars #5722CD;
+Blauw #4845E4;
+Roze #ED4385;
+Zwart #0C0C0C (voor standaard tekst);
+Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services")
+Uitzonderingen: Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)
+Grijs #7A7A7A (voor de service-tegel tekst);
+/////////////////////////////////////////////////////////*/
+
+/*---------------------------------1. global styles----------------------------------------*/
+
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --lightpurple: #F2F0FF;
+    --blue: #4845E4;
+}
+
+/*---------------------------------2. typography----------------------------------------*/
+
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #0C0C0C;
+}
+
+h1, h2, h3, h4 {
+    font-family: 'Merriweather', serif;
+}
+
+a:link {
+    text-decoration: none;
+    color: white;
+}
+
+/*---------------------------------3. areas----------------------------------------*/
+
+/*---------------------------------3.1 header----------------------------------------*/
+header {
+    background-color: var(--purple);
+    background-image: url("/assets/noise-texture.png"), linear-gradient(to right, var(--blue), hsl(259, 72%, 47%));
+    color: #F3F6F9;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+}
+
+nav {
+    width: 100vw;
+    max-width: 100%;
+}
+
+nav ul {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+}
+
+nav li {
+    margin: 20px 30px 20px 30px;
+}
+
+.intro {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 60px 0 0 0;
```

Als die waarden 0 zijn, en niet gedaan is om een andere waarde te overschrijven, dan zie ik liever:
```suggestion
    padding-top: 60px;
```